spec/classes/sensu_init_spec.rb

```diff
@@ -402,7 +402,7 @@
           context "when #{var_name} (#{type}) is set to invalid #{invalid} (as #{invalid.class})" do
             let(:params) { [mandatory_params, var[:params], { :"#{var_name}" => invalid, }].reduce(:merge) }
             it 'should fail' do
-              expect { should contain_class(subject) }.to raise_error(Puppet::Error, /#{var[:message]}/)
+              expect { should contain_class(subject) }.to raise_error(Puppet::PreformattedError)
```

Comment:
@alvagante I still like to test that input parameter validation is as expected. Think this makes even sense with Puppet v4 and the new data type validations. It is not mandatory to have validation in Puppet v4 so it's still a module feature to take care for.

Testing for the actual error messages does proof that there is no other error popping up. With the general ```PreformattedError``` it would accept any error message regardless of the real reason.